### ISSUE TITLE
change for new map and better location descriptions per-page

### ISSUE THREAD

**Issue by kir12** [kir12/touhoufest-webapp#34]
with the introduced new map, some event types have had their location names changed, and connections between the map's room labels and the rooms' actual names need to be established to avoid confusion.

- in the maps page, replace the 2023 maps with the new 2024 map, and add the new artist alley map ✅
- below the map, add a list of labels by number and have next to each number the actual name that the con center has for it as dictated by MikoTan. the ones he's given so far are below; for the rest either look it up or ask someone ✅

```
2 - George Nakano Theatre
3/5/6 - Entry Plaza
4 - Garden Rooms A/B
15 - Pine Wind Garden
9 - Toyota Meeting Hall
10/11 - Ken Miller Rec Center
12 - Entry Foyer
13 - Assembly Hall
7 - Torino Festival Plaza
```

- adjust the key locations page with updated pictures to each of these locations and remove unneeded pictures ✅
- inside each of the hobby pages, add the corresponding number next to each bolded location as well as a *description of where it is relative to other areas of the convention* ❌

**Comment by kir12** [comment]
depends on #33

**Comment by kir12** [comment]
with #33 now closed this issue is free to proceed

**Comment by kir12** [comment]
dropping the numbered indicator as multiple numbers can be attributed to a location. if someone asks for it it'll be part of a new issue

**Comment by kir12** [comment]
gaming's location needs to be updated inside #20. The main branch will be merged into the gaming branch to bring its changes over, at which point the gaming location stuff will also be cleaned up.
